acir/src/native_types/arithmetic.rs

```diff
@@ -149,6 +149,20 @@ impl Expression {
         Ok(expr)
     }
 
+    // Returns one witness belonging to the expression, in no relevant order
+    // Returns None if the expression is const
+    pub fn any_witness(&self) -> Option<Witness> {
```

Comment:
Can you put a brief comment on why this is needed? It seems like its just for when we return Errors?

Not blocking for this PR, but I think we can refactor it later so that its not in the namespace of arithmetic and is instead a free-standing function in the module that needs it

Reply:
It is not needed strictly speaking, I use it when evaluating an expression during solving to report one non-solved witness. It could be move to the pwg part. I have added a comment on its usage.

Reply:
A comment is fine, would be better to move it to where its needed

Reply:
I can move it to pwg

Reply:
Done